scan/lib/scan/options.rb

```diff
@@ -31,7 +31,6 @@ def self.available_options
                                      optional: true,
                                      env_name: "SCAN_PROJECT",
                                      description: "Path to the project file",
-                                     conflicting_options: [:package_path],
```

Comment:
Hello, thanks for the contribution. Why remove this line?

Reply:
☝️